### PR TITLE
Remove TOU and PP summary, require acceptance of new terms when they are updated

### PR DESCRIPTION
**Pull request by pazaan**
Add support for an environment variable called `LATEST_TERMS` which should be a string that is parsable by JavaScript Date(), preferably in ISO 8601 combined format so as to be unambiguous.
This should be the date of the most recent Tidepool Terms of Use. When the date is more recent than a users’ ToU acceptance date, the ToU terms page will be shown and the user will need to re-accept the terms before being able to continue to the Tidepool web app.

### CODE REVIEW

**Comment by pazaan** [comment]
[Remove TOU and PP summary, update date, require acceptance of new terms](https://trello.com/c/gHVbaisM/162-remove-tou-and-pp-summary-update-date-require-acceptance-of-new-terms)

**Comment by jh-bate** [comment]
@pazaan looks like the tests are failing:

see `https://travis-ci.org/tidepool-org/blip/jobs/224684043#L2302`

**Comment by jebeck** [comment]
I just had a few minutes to check in. @pazaan the tests are failing because you have not based this work on the node update. As soon as the node update #405 is approved for prod it needs to be merged to master and this work needs to merge in the update from master. You may also have failing tests after that, as it doesn't look like you've updated any tests yet and there _should_ be existing tests around both the personutils and the terms "page."

File: app/config.js [hunk 1]
```diff
@@ -15,9 +15,13 @@
 
 /* global __TEST__ */
 
+if (__TEST__ && (window === undefined || window.config === undefined)) {
```
Comment (jh-bate) [inline app/config.js]: @jebeck this was the update that allowed the tests to work. `personutils.js` now has a reference to config and that was killing the tests.

**Comment by jh-bate** [comment]
approved so now merging

**Pull request merged**